requirements.txt

```diff
@@ -11,3 +11,5 @@ PyYAML>=5.3.1
 scikit-image>=0.15.0
 tensorboardX>=1.8
 pandas>=1.1.5
+numpy==1.19.5
+tensorflow==2.4.1
```

Comment:
Why should we have an explicit dependency on TensorFlow in the patch?

Reply:
We had pushed by mistake for testing, shall remove the same.

Reply:
I suppose, it is the same problem as in #253. Let's fix it in another PR.

Reply:
Fixed in #278